packages/core/bootstrap/src/lib/cache-warmer/epics.ts

```diff
@@ -174,18 +174,44 @@ export const warmupRequestHandler: Epic<AnyAction, AnyAction, any> = (action$, s
     filter(warmupRequested.match),
     // fetch our required state to make a request to warm up an adapter
     withLatestFrom(state$),
-    map(([action, state]) => ({
-      requestData: state.cacheWarmer.subscriptions[action.payload.key],
-      key: action.payload.key,
-    })),
+    map(([action, state]) => {
+      const requestData = state.cacheWarmer.subscriptions[action.payload.key]
+      let batchablePropertyPath
+      if (requestData && requestData.childLastSeenById && Object.keys(requestData.childLastSeenById).length > 0) {
+        const child = state.cacheWarmer.subscriptions[Object.keys(requestData.childLastSeenById)[0]]
+        batchablePropertyPath = child.batchablePropertyPath
+      }
+      return {
+        requestData: {
+          ...requestData,
+          batchablePropertyPath
+        },
+        key: action.payload.key,
+        subscriptions: state.cacheWarmer.subscriptions 
+      }
+    }),
     filter(({ requestData }) => !!requestData),
     // make the request
-    mergeMap(({ requestData, key }) =>
+    mergeMap(({ requestData, key, subscriptions }) =>
       from(
-        requestData.executeFn({
-          id: requestData.childLastSeenById ? WARMUP_BATCH_REQUEST_ID : WARMUP_REQUEST_ID,
-          data: { ...requestData.origin, maxAge: -1 },
-        }),
+        (async () => {
+          const batches = splitIntoBatches(requestData)
+          let result = null 
+          for (const batch of Object.values(batches)) {
+            const data = {
+              ...requestData.origin,
+              ...batch,
+              resultPath: getBatchRequestResultPath(requestData.batchablePropertyPath, subscriptions, batch),
+              maxAge: -1
+            }
+            const batchResult = await requestData.executeFn({
```

Comment:
We should send these in parallel rather than awaiting each response with something like `Promise.all`.